src/wwwroot/app/views/vacancy.profile/vacancy.view.controller.js

```diff
@@ -0,0 +1,116 @@
+const LIST_OF_THESAURUS = ['stages', 'industries'];
+import {
+   remove,
+   set,
+   each
+} from 'lodash';
+
+export default function VacancyProfileController(
+   $scope,
+   $state,
+   $translate,
+   $element,
+   ThesaurusService,
+   UserService,
+   VacancyService,
+   UserDialogService,
+   FileService,
+   LoggerService
+   ) {
+   'ngInject';
+
+   const vm = $scope;
+   vm.thesaurus    = [];
+   vm.responsibles = [];
+   vm.edit         = edit;
+   vm.uploader     = createNewUploader();
+   vm.addFilesForRemove = addFilesForRemove;
```

Comment:
Because we cannot remove saved files from database immediately. So I create a queue of files and after that remove this files on click 'Submit'.